eo-maven-plugin/src/test/java/org/eolang/maven/FakeMaven.java

```diff
@@ -430,6 +466,19 @@ private static Set<String> mojoFields(final Class<?> mojo, final Set<String> fie
         return res;
     }
 
+    /**
+     * Fill up given tojos by attributes.
+     * @param tojos Tojos to fill up.
+     */
+    private void fillUpTojos(final List<Tojo> tojos) {
```

Comment:
@maxonfjvipon Please, rename that method according to that article: https://www.yegor256.com/2015/01/12/compound-name-is-code-smell.html

Moreover, I would suggest to move that method into `ForeignTojos`:
https://www.yegor256.com/2016/04/05/printers-instead-of-getters.html
Let's treat `ForeignTojos` as an object (not as a collection of data) and give it a chance to fulfill itself. I'm sure it knows better how to do so :)

Reply:
@volodya-lombrozo what would you suggest to name this method?

Reply:
@maxonfjvipon something like `withAttributes`

Reply:
@volodya-lombrozo I don't see a fast way to move the method into `ForeignTojos` because `ForeignTojo` does not have a method `set`. So I'll just rename it and add the puzzle.